src/Evolution/DgSubcell/Actions/SelectNumericalMethod.hpp

```diff
@@ -0,0 +1,67 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <tuple>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Evolution/DgSubcell/Actions/Labels.hpp"
+#include "Evolution/DgSubcell/ActiveGrid.hpp"
+#include "Evolution/DgSubcell/Tags/ActiveGrid.hpp"
+#include "Parallel/Actions/Goto.hpp"
+#include "Utilities/ErrorHandling/Error.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace Parallel {
+template <typename Metavariables>
+class GlobalCache;
+}  // namespace Parallel
+namespace tuples {
+template <typename...>
+class TaggedTuple;
+}  // namespace tuples
+/// \endcond
+
+namespace evolution::dg::subcell::Actions {
+/*!
+ * \brief Goes to `Labels::BeginDg` or `Labels::BeginSubcell` depending on
+ * whether the active grid is `Dg` or `Subcell`.
+ *
+ * GlobalCache:
+ *
+ * DataBox:
+ * - Uses:
+ *   - `subcell::Tags::ActiveGrid`
```

Comment:
This formatting is confusing (the empty GlobalCache line in particular). I think it would be easier to understand a "verbs first" formatting that looks something like this...
```
Uses:
- GlobalCache: nothing
- DataBox:
  - tags
```

Reply:
I'm happy to add the `nothing` but the other pattern I think is what we have been using on all actions throughout the code base so I'd rather not start changing that just in one place now